jax/_src/test_util.py

```diff
@@ -1198,15 +1198,6 @@ def strict_promotion_if_dtypes_match(dtypes):
     return jax.numpy_dtype_promotion('strict')
   return jax.numpy_dtype_promotion('standard')
 
-_version_regex = re.compile(r"([0-9]+(?:\.[0-9]+)*)(?:(rc|dev).*)?")
-def _parse_version(v: str) -> tuple[int, ...]:
-  m = _version_regex.match(v)
-  if m is None:
-    raise ValueError(f"Unable to parse version '{v}'")
-  return tuple(int(x) for x in m.group(1).split('.'))
-
-def numpy_version():
-  return _parse_version(np.__version__)
```

Comment:
This no longer has to be removed from `test_util.py`, since it's not being used elsewhere, correct?

I'd prefer to keep it here because it makes it clear that versions checks are a thing that should only be used in tests.

Reply:
Ah, sure! It's again used only in tests - I revered it.